### PR TITLE
Wasm cpp bazel

### PR DESCRIPTION
**Pull request by Hinsteny**
### Ⅰ. Describe what this PR did
support make build cpp wasmplugin and do e2e test

### Ⅱ. Does this pull request fix one issue?
NO

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
has e2e test case

### Ⅳ. Describe how to verify it
PLUGIN_TYPE=CPP PLUGIN_NAME=key_auth make higress-wasmplugin-test

### Ⅴ. Special notes for reviews
NO

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -29,6 +30,8 @@ import (
 )
 
 var isWasmPluginTest = flag.Bool("isWasmPluginTest", false, "")
+var wasmPluginType = flag.String("WasmPluginType", "GO", "")
+var wasmPluginName = flag.String("WasmPluginName", "", "")
```
Comment (CH3CHO) [inline test/e2e/e2e_test.go:34]: 新加的参数为啥是首字母大写呢？

Comment (johnlanni) [reply test/e2e/e2e_test.go:34]: @Hinsteny 请保持一致

**Review by johnlanni — CHANGES_REQUESTED** [review]

**Comment by Hinsteny** [comment]
@CH3CHO @johnlanni Just change the e2e_test file param name start with lowercase, pls help me RE again.

**Review by johnlanni — APPROVED** [review]
LGTM

**Pull request merged**